lib/posix/fs.c

```diff
@@ -63,8 +95,13 @@ int open(const char *name, int flags, ...)
 {
 	int rc, fd;
 	struct posix_fs_desc *ptr = NULL;
+	int zmode;
```

Comment:
Where possible initialize a variable when it's declared: make this:

    int zmode = posix_mode_to_zephyr(flags)

Unless @pfalcon objects.

Reply:
Done.